packaging/fetch_gpg_key/main.go

```diff
@@ -0,0 +1,108 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+// The application fetches encrypted certificate from GCS bucket,
+// decrypts the certificate using cloudkms and stores it in /tmps
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+
+	cloudkms "cloud.google.com/go/kms/apiv1"
+	"cloud.google.com/go/storage"
+	"github.com/kylelemons/godebug/pretty"
+	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
+)
+
+// The values are used for testing only. The values will be replaced with actual ones
+// before merging the commit
+var (
+	dump = &pretty.Config{IncludeUnexported: true}
+
+	gcsBucket = "signing-certificate"
+	gcsObject = "RPM-GPG-KEY-subrat.encrypted"
+
+	projectID   = flag.String("gcs_project", "subratp-project", "GCS project where the certificate is stored")
+	keyRingID   = flag.String("key_ring", "subratp-testing-gpg", "key ring used for decryption")
+	keyName     = flag.String("key_name", "rpm-gpg", "key name used for decryption")
+	keyLocation = flag.String("location", "global", "GCP location where the kms key is stored")
+	kmsKeyName  = fmt.Sprintf("projects/%s/locations/%s/keyRings/%s/cryptoKeys/%s", *projectID, *keyLocation, *keyRingID, *keyName)
+	outFileName = flag.String("out_path", "/tmp/RPM-GPG-KEY-subrat-decrypted", "output file path name")
+)
+
+func main() {
+	ctx := context.Background()
+	fmt.Println("reading encrpyted file...")
+	dat, err := readFromGcs(ctx, gcsBucket, gcsObject)
+	check(err)
+	fmt.Printf("decrypting file...\n")
+	ddata, err := decrypt(ctx, kmsKeyName, dat)
+	check(err)
+	fmt.Println("writing data to file")
+	writeFile(ddata, "/tmp/RPM-GPG-KEY-subrat-decrypted")
+	fmt.Println("stored decrypted signature file")
+}
+
+func writeFile(data *[]byte, outFilePath string) {
+	err := ioutil.WriteFile(outFilePath, *data, 0400)
+	check(err)
```

Comment:
This function should be returning an error, but since it's just using WriteFile whats the point?